config/settings_data.json

```diff
@@ -1,5 +1,83 @@
 {
-  "current": "Default",
```

Comment:
I apologize, I made some changes to test some things and saved so youll have to revert closer to merging

Reply:
Np reverted now and used a duplicate store in the PR description instead :)